lighthouse-core/lib/task-groups.js

```diff
@@ -5,90 +5,108 @@
-  'Frame Start': groupIdToName.painting,
-  'Frame Start (main thread)': groupIdToName.painting,
-  'Cancel Animation Frame': groupIdToName.scripting,
-  'Cancel Idle Callback': groupIdToName.scripting,
-  'Compile Script': groupIdToName.scriptParseCompile,
-  'Composite Layers': groupIdToName.compositing,
-  'Console Time': groupIdToName.scripting,
-  'Image Decode': groupIdToName.images,
-  'Draw Frame': groupIdToName.painting,
-  'Embedder Callback': groupIdToName.scripting,
-  'Evaluate Script': groupIdToName.scripting,
-  'Event': groupIdToName.scripting,
-  'Animation Frame Fired': groupIdToName.scripting,
-  'Fire Idle Callback': groupIdToName.scripting,
-  'Function Call': groupIdToName.scripting,
-  'DOM GC': groupIdToName.scriptGC,
-  'GC Event': groupIdToName.scriptGC,
-  'GPU': groupIdToName.gpu,
-  'Hit Test': groupIdToName.compositing,
-  'Invalidate Layout': groupIdToName.styleLayout,
-  'JS Frame': groupIdToName.scripting,
-  'Input Latency': groupIdToName.scripting,
-  'Layout': groupIdToName.styleLayout,
-  'Major GC': groupIdToName.scriptGC,
-  'DOMContentLoaded event': groupIdToName.scripting,
-  'First paint': groupIdToName.painting,
-  'FMP': groupIdToName.painting,
-  'FMP candidate': groupIdToName.painting,
-  'Load event': groupIdToName.scripting,
-  'Minor GC': groupIdToName.scriptGC,
-  'Paint': groupIdToName.painting,
-  'Paint Image': groupIdToName.images,
-  'Paint Setup': groupIdToName.painting,
-  'Parse Stylesheet': groupIdToName.parseHTML,
-  'Parse HTML': groupIdToName.parseHTML,
-  'Parse Script': groupIdToName.scriptParseCompile,
-  'Other': groupIdToName.other,
-  'Rasterize Paint': groupIdToName.painting,
-  'Recalculate Style': groupIdToName.styleLayout,
-  'Request Animation Frame': groupIdToName.scripting,
-  'Request Idle Callback': groupIdToName.scripting,
-  'Request Main Thread Frame': groupIdToName.painting,
-  'Image Resize': groupIdToName.images,
-  'Finish Loading': groupIdToName.loading,
-  'Receive Data': groupIdToName.loading,
-  'Receive Response': groupIdToName.loading,
-  'Send Request': groupIdToName.loading,
-  'Run Microtasks': groupIdToName.scripting,
-  'Schedule Style Recalculation': groupIdToName.styleLayout,
-  'Scroll': groupIdToName.compositing,
-  'Task': groupIdToName.other,
-  'Timer Fired': groupIdToName.scripting,
-  'Install Timer': groupIdToName.scripting,
-  'Remove Timer': groupIdToName.scripting,
-  'Timestamp': groupIdToName.scripting,
-  'Update Layer': groupIdToName.compositing,
-  'Update Layer Tree': groupIdToName.compositing,
-  'User Timing': groupIdToName.scripting,
-  'Create WebSocket': groupIdToName.scripting,
-  'Destroy WebSocket': groupIdToName.scripting,
-  'Receive WebSocket Handshake': groupIdToName.scripting,
-  'Send WebSocket Handshake': groupIdToName.scripting,
-  'XHR Load': groupIdToName.scripting,
-  'XHR Ready State Change': groupIdToName.scripting,
+/**
+ * Make sure the traceEventNames keep up with the ones in DevTools
```

Comment:
Well the whole list is new :) last time we were keying off the display names, I pulled everything I saw that I recognized/looked like it could be substantial